manager/integration/tests/test_ha.py

```diff
@@ -635,3 +636,83 @@ def test_rebuild_failure_with_intensive_data(client, core_api, volume_name, pod_
     delete_and_wait_pod(core_api, pod_name)
     delete_and_wait_pvc(core_api, pvc_name)
     delete_and_wait_pv(core_api, pv_name)
+
+
+def test_rebuild_duplicate_replica(client, core_api, volume_name, pod_make):  # NOQA
```

Comment:
`duplicate_replica` part in the name is confusing.

Be more specific, e.g. `test_rebuild_from_replica_field`.

Also, do we need a separate test case for the field? We can add it in the existing replica rebuild test.

Reply:
This test is not only for the field `fromReplica, but also for the corner case that the from-replica and the rebuilding replica are on the same node.

Reply:
Make sense. Rename the test case to show that should be good enough.